libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFSDKViewUtils.m

```diff
@@ -26,36 +26,35 @@
  WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 #import "SFSDKViewUtils.h"
-#import "SFSDKLoginViewControllerConfig.h"
+#import "SFSDKViewControllerConfig.h"
 
 @implementation SFSDKViewUtils
 
-+ (void)styleNavigationBar:(UINavigationBar *)navigationBar config:(SFSDKLoginViewControllerConfig *) config {
-    
++ (void)styleNavigationBar:(UINavigationBar *)navigationBar config:(SFSDKViewControllerConfig *) config {
+
     if (!navigationBar && !config) {
         return;
     }
     
     if (config.navBarColor) {
-        UIImage *backgroundImage = [self headerBackgroundImage:config.navBarColor];
         if (@available(iOS 13.0, *)) {
-            navigationBar.standardAppearance.backgroundImage = backgroundImage;
+            navigationBar.standardAppearance.backgroundColor = config.navBarColor;
```

Comment:
Image doesn’t get updated on dark/light transition if the nav bar color is a dynamic color but works automatically with background color. I tested it with the login and passcode nav bars on a handful devices, let me know if there are other scenarios I should check

Reply:
Great catch!